hub/core/compression.py

```diff
@@ -4,13 +4,22 @@
     SampleDecompressionError,
     UnsupportedCompressionError,
 )
-from typing import Union
+from typing import Union, Tuple
 import numpy as np
 
 from PIL import Image, UnidentifiedImageError  # type: ignore
 from io import BytesIO
 
 
+def to_image(array: np.ndarray) -> Image:
+    shape = array.shape
+    if len(shape) == 3 and shape[0] != 1 and shape[2] == 1:
```

Comment:
is `shape[0] != 1` necessary / accurate? Width can be technically 1 right?

Reply:
@farizrahman4u the issue in this case is that we are unsure of the layout: if it's (1,N,1) you don't know if the channels are the first or last dimension, and so cannot make any assumptions. It's not okay to just assume the channels come last because if they don't follow this layout then they'll get data in the wrong shape [(1,N) instead of (N,1)].

Reply:
Ah makes sense, thanks!